data.js

```diff
@@ -29,23 +29,26 @@ export const showAllData = function (resultFetchBreed, resultDataUl, newLi) {
   resultDataUl.innerHTML = "";
   if (resultFetchBreed === undefined) {
   } else {
-    for (let i = 0; i < resultFetchBreed.length; i++) {
-      newLi = document.createElement("li");
-      newLi.classList.add("result");
-      newLi.innerHTML = `<div class="result__num">${
-        i + 1
-      }</div><div class ="result__breedLink">${
-        resultFetchBreed[i].name
-      }</div><img class="hidden" src="${
-        resultFetchBreed[i].image.url
-      }" alt="Image of the ${resultFetchBreed[i].name}">`;
-
-      resultDataUl.append(newLi);
-    }
+    addDataToHtml(resultFetchBreed, newLi, resultDataUl);
   }
 };
 
 export const clearData = function (searchInput) {
   inputFocus();
   searchInput.value = "";
 };
+
+export const addDataToHtml = function (fetchResults, newLi, resultDataUl) {
+  for (let i = 0; i < fetchResults.length; i++) {
+    newLi = document.createElement("li");
+    newLi.classList.add("result");
+    newLi.innerHTML = `<div class="result__num">${
+      i + 1
+    }</div><div class ="result__breedLink">${
+      fetchResults[i].name
+    }</div><img class="hidden" src="${
+      fetchResults[i].image.url
```

Comment:
Sedaj mi ne deluje prikaz slik, če npr išem določeno pasmo, ker v tej AddDataToHtml fn preko newLi dodam html tags-e. Vendar pa bi moral sedaj pri tej fn dodat dynamic url za src image, ker tako funkcija showAllData in pa fn searchBreed pokličeta to fn, vendar pa obedvi fn uporabljata drugačen url. 
Obedvi loopata čez podatke , kot npr tukaj `fetchResults[i].image.url`, vendar pa ena uporablja ta `image.url`, druga pa `reference_image_id`, se pravi `fetchResults[i].image.id`, druga pa `resolve[i].reference_image_id`. Sedaj pa ne vem kako nastavit ta dinamičen url v tej AddDataToHtml ,da jo lahko uporabljam kjerkoli, ker to ja namen funkcij.  Maš kako usmeritev

Reply:
lahko si sestaviš to strukturo po kateri `addDataToHtml` loopa tako, da so notr samo podatki k jih rabiš za sestavt html.. sepravi, ko kličeš `addDataToHtml` iz ene funkcije passaš notr to kar takrat rabiš, ko iz druge pa drugač.. 
strukturo `fetchResults` arraya mislem.. ok?
recimo da je to ta array
`[{
name: 'Filip',
url: 'nekineki'
}]`

in si preden pokličeš to funkcijo to pravilno strukturiraš pa je.. in s tem tud ubistvu funkcijo sploh ne zanima kaj je notr.. ona samo gre čez in sestavi html..